src/components/UI/AppDetails/AppDetails.js

```diff
@@ -88,8 +95,62 @@ const Install = styled.div`
   }
 `;
 
+const ChartVersionTable = styled.table`
+  border: 1px solid ${props => props.theme.colors.shade4};
+  margin-top: 10px;
+  max-width: 320px;
+  table-layout: fixed;
+  white-space: nowrap;
+
+  td {
+    vertical-align: top;
+    code {
+      ${Ellipsis}
+      max-width: 150px;
+      display: inline-block;
+    }
+
+    .copyable {
```

Comment:
This references a className that is assigned in a different component. If the `Copyable` component changes its structure, this will not work in the same way.

Would you consider extending `Copyable`?
Ex:
```javascript
const StyledCopyable = styled(Copyable)`
  ...styles
`
```

And then using it as normal: `<StyledCopyable />`.

(Please note that in order to do this, the `className` prop must be able to be passed to the root element inside the `Copyable` component, so passing the `className` prop to the root `div` inside `Copyable` should do the job)

Reply:
The thing is I want to target a copyable inside a certain other element, and not the other copyables.

I could make two different StyleCopyables, but is there a better way?

I was looking at this but I can't seem to make it work: https://emotion.sh/docs/styled#targeting-another-emotion-component

Reply:
Yes, that is actually a good idea, and it should work, since we added the babel transformer for emotion.

Don't forget to pass the `className` prop, so it could work
Just need to do this:
![image](https://user-images.githubusercontent.com/13508038/75798276-a260d300-5d76-11ea-9159-d094f1430ec9.png)

Reply:
Somehow can't get it to work, will go for two styled `Copyable`s instead.

The error I got was `invalid hook call`, so it's like it's trying to render the component somehow.

Reply:
Ahh yes, forgot about this. To make it work:
- Pass the `className` property to the root component of `Copyable`
- First, define a styled copy of `Copyable`
```javascript
const StyledCopyable = styled(Copyable)``
```

And then target that one inside other styled components
```javascript
${Copyable} {
  ...styles
}
```

Reply:
It works! w00t